Changes

```diff
@@ -231,6 +231,12 @@ Working version
   (Vincent Laviron, review by Pierre Chambart and Leo White,
    report by Aleksandr Kuzmenko)
 
+- #8955: Allow compiling ocamldoc with -principal by fixing inheritance
+  (Jacques Garrigue)
+
+OCaml 4.09.0
+------------
+
```

Comment:
That looks like a mistake.

Reply:
Indeed, bad resolution of conflict, this is why the diff looked strange.